### PR TITLE
[Experimental] Self-contained deployment samples

### PR DESCRIPTION
**Pull request by mikebattista**
## Description

Self-contained deployment is in scope for 1.1. These are early experimental samples demonstrating self-contained deployment. Will expand to more languages and project types once the general pattern for the sample is reviewed and approved.

## Checklist

- [x] Samples build and run using the Visual Studio versions listed in the [Windows development docs](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=stable#2-install-visual-studio).
- [x] Samples build and run on all supported platforms (x64, x86, ARM64) and configurations (Debug, Release). 
- [x] Samples set the minimum supported OS version to Windows 10 version 1809.
- [x] Samples build clean with no warnings or errors.
- [x] **[For new samples]**: Samples have completed the [sample guidelines checklist](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#checklist) and follow [standardization/naming guidelines](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#standardization-and-naming).

### CODE REVIEW

**Comment by mikebattista** [comment]
@angelazhangmsft can we merge this PR?

**Review by btueffers — APPROVED** [review]

File: Samples/SelfContainedDeployment/cpp-winui-packaged/packages.config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Windows.CppWinRT" version="2.0.210922.5" targetFramework="native" />
+  <package id="Microsoft.Windows.SDK.BuildTools" version="10.0.22000.194" targetFramework="native" />
+  <package id="Microsoft.WindowsAppSDK" version="1.1.0-20220312.0-CI-experimental" targetFramework="native" />
```
[the remark this answers is not in the document]
Comment (mikebattista) [reply Samples/SelfContainedDeployment/cpp-winui-packaged/packages.config]: Updated all samples to preview3.

**Pull request merged**